MiniTeks/mks-cli
# Create command-line interface to interact with mks-server

##  Create mks-cli to interact with mksServer.
##### Available Commands:- "$ ./mks-cli \<cmd-name>"
  * create -     ceates mkstask
  * delete -    deletes mkstask
  * get -        gets mkstasks
  * list -       lists mkstasks

##### To test:
1. Build and execute mks-server. (https://github.com/MiniTeks/mks-server)
2. Build and install mks-cli (see README.md)
3. execute your command, e.g- 
```bash
                   ./mks-cli mkstask get <mkstaskname> -n <namespace>
``` 

Fixes #3
